### ISSUE TITLE
Should the manual document `seL4_GetMR`/`seL4_SetMR`?

### ISSUE THREAD

**Issue by Ivan-Velickovic** [seL4/seL4#1100]
The libsel4 `seL4_GetMR` and `seL4_SetMR` wrappers over accessing the IPC buffer are not documented in the manual.

I can understand not wanting to couple the manual to libsel4 since it's just a wrapper over the actual system calls and hence not include these two functions. However, the manual is already coupled to libsel4.

**Comment by lsf37** [comment]
Yes, it's basically a libsel4 manual, I think it would make sense to include those functions.
